src/module/apps/compendium-browser/index.ts

```diff
@@ -272,44 +296,210 @@ class CompendiumBrowser extends Application {
+                        console.error("Initital filtering for ranges other than price aren't implemented yet.");
+                        continue;
+                    }
+
+                    // Set max value to min value if min value is higher
+                    if (range.values.min > range.values.max) {
+                        range.values.max = range.values.min;
+                        range.values.inputMax = range.values.inputMin;
                     }
+
+                    range.isExpanded = true;
                 }
             }
-            this.initialFilter = [];
-            this.initialMaxLevel = 0;
+            // Sliders (e.g. level)
+            else if (
+                currentTab.filterData.sliders !== undefined &&
+                objectHasKey(currentTab.filterData.sliders, mappedFilterType) &&
+                this.isRange(filterValue)
+            ) {
+                if (
+                    (filterValue.min !== undefined && filterValue.min !== null) ||
+                    (filterValue.max !== undefined && filterValue.max !== null)
+                ) {
+                    const slider = currentTab.filterData.sliders[mappedFilterType];
+
+                    let minValue = typeof filterValue.min === "string" ? parseInt(filterValue.min) : filterValue.min;
+                    let maxValue = typeof filterValue.max === "string" ? parseInt(filterValue.max) : filterValue.max;
+                    if ((minValue && isNaN(minValue)) || (maxValue && isNaN(maxValue))) {
+                        console.error(`Invalid filter value for '${filterType}', it needs to be a valid number.`);
+                        continue;
+                    }
+
+                    minValue =
+                        minValue !== undefined && minValue !== null
+                            ? Math.clamped(minValue, slider.values.lowerLimit, slider.values.upperLimit)
+                            : slider.values.lowerLimit;
+                    maxValue =
+                        maxValue !== undefined && maxValue !== null
+                            ? Math.clamped(maxValue, slider.values.lowerLimit, slider.values.upperLimit)
+                            : slider.values.upperLimit;
+
+                    // Set max value to min value if min value is higher
+                    if (minValue > maxValue) {
+                        maxValue = minValue;
+                    }
+
+                    slider.values.min = minValue;
+                    slider.values.max = maxValue;
+                    slider.isExpanded = true;
+                }
+            }
+            // Filter name did not match a filter on the tab
+            else {
+                console.warn(`'${filterType}' is not a valid filter for tab '${currentTab.tabName}'.`);
+            }
         }
 
-        this.render(true);
+        this.initialFilter = {};
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
avoid disabling the linter for any. Instead leave value as type unknown and take advantage of typeof and the isObject() helper function.